src/App.js

```diff
@@ -5,7 +5,13 @@ import BooksForm from './containers/BooksForm';
 function App() {
   return (
     <div>
-      <h1>Magic Books</h1>
+      <header className="">
+        <h1 id="home">Magic Books CMS</h1>
+        <nav>
+          <a href="#home">BOOKS</a>
+          <a href="#home">CATEGORIES</a>
+        </nav>
+      </header>
```

Comment:
> However, moving the Categories filter into the nav section isn't practicable for me now because the CategoryFilter component receives props from its parent (the BooksList component). To do that will affect the current setup of the App.

You can move this to the `BooksList` component, by doing that the `CategoryFilter` component can be used in place of `<a href="#home">CATEGORIES</a>` as suggested by @Oghenebrume50